components/engine_traits/src/import.rs

```diff
@@ -34,3 +28,9 @@ pub trait Import: CFHandleExt {
         expected_checksum: u32,
     ) -> Result<()>;
 }
+
+pub trait IngestExternalFileOptions {
```

Comment:
Shall we convert other options to traits to keep consistency in this PR?

Reply:
Maybe step by step is better.